book/07-git-tools/sections/advanced-merging.asc

```diff
@@ -351,25 +351,25 @@ end
 hello()
 ----
 
-Both sides of the merge added content to this file, but some of the commits modified the file in the same place that caused this conflict.
+Bawat panig sa merge ay nagdaragdag ng nilalaman sa file na ito, ngunit marami sa mga commit na nagbago sa file sa parehong lugar na sanhi sa pagkasalungat nito.
 
-Let's explore a couple of tools that you now have at your disposal to determine how this conflict came to be.
-Perhaps it's not obvious how exactly you should fix this conflict.
-You need more context.
+Talakayin natin ang karamihan sa mga kasangkapan na sa ngayon ay alam mo na iyong magagamit upang malaman kung papaano ang pagkasalungat ay dumating.
+Marahil ito ay hindi halata kung gaano ka eksakto na dapat mong ayusin ang salungat.
+Kailangan mo ng maraming konteksto.
 
-One helpful tool is `git checkout` with the `--conflict' option.
-This will re-checkout the file again and replace the merge conflict markers.
-This can be useful if you want to reset the markers and try to resolve them again.
+Isang matulugin na kasangkapan ay ang `git checkout` kasama ang `--conflict' na opsyon.
+Ito ay magre-checkout muli sa file at papalitan ang salungat na merge na mga marka.
```

Comment:
``magre-checkout`` to ``mag-checkout muli``

Reply:
change done